src/funcs.c

```diff
@@ -2825,3 +2825,71 @@ void joy_right()
   set(app->LV_GamesList, MUIA_List_Active, ind-1);
 
 }
+
+ULONG get_wb_version()
+{
+	static ULONG ver = 0UL;
+
+	if (ver != 0UL)
+	{
+		return ver;
+	}
+
+	if (WorkbenchBase == NULL)
+	{
+		//Really Workbench library should be already opened
+		// Somehow we're running without Workbench.
+		// Nothing to do since the version variable inits with 0.
+		return ver;
+	}
+
+	// Save workbench.library version for later calls
+	ver = WorkbenchBase->lib_Version;
+
+	return ver;
+}
+
+void open_current_dir()
+{
+	if (get_wb_version() < 44)
+	{
+		// workbench.library doesn't support OpenWorkbenchObjectA yet
+		return;
+	}
+
+	// Allocate Memory for variables
+	char* game_title = NULL;
```

Comment:
Can you please move it at the top of the function?

Reply:
Changed in a01ea87 (to be pushed)